common/djangoapps/util/url.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility functions related to urls.
+"""
+
+import sys
+from django.conf import settings
```

Comment:
There should be a line of whitespace between standard library imports and third-party library imports